src/js_links.hpp

```diff
@@ -65,28 +65,32 @@ template <typename T>
 class MixedLink : public MixedWrapper<typename T::Context, typename T::Value> {
   private:
     using Context = typename T::Context;
-    using Value = typename T::Value;
+    using ValueType = typename T::Value;
+    using Value = js::Value<T>;
     using RealmClass =  RealmObjectClass<T>;
 
     std::shared_ptr<Realm> realm;
 
   public:
     MixedLink(std::shared_ptr<Realm> _realm): realm{_realm} {}
 
-    Mixed wrap(Context context, Value const& value) {
+    Mixed wrap(Context context, ValueType const& value) {
+
+        if (Value::is_array(context, value)) {
+            throw std::runtime_error("An array cannot be converted to Mixed.");
```

Comment:
Technically, nothing is being "converted" here.
From the endusers point of view, they're trying to store an array as the value of a `Mixed` property.
Which is why I suggested the following message:
> A mixed property doesn't support lists of values

Perhaps "support" is not as precise as we want to be. Another alternative could be:

> Arrays are not a mixed value

or

> Cannot use an array as a Mixed value

Reply:
Or maybe `A mixed property cannot contain an array of values`?